packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -300,4 +328,6 @@ export class LicenseManager {
 			`color: white; background: crimson; padding: 2px; border-radius: 3px;`
 		)
 	}
+
+	static classname = 'tl-watermark_SEE-LICENSE'
```

Comment:
nit: might as well make this consistent with react-type casing to be `className`